esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -1125,7 +1125,7 @@ unsigned WsWuInfo::getWorkunitThorLogInfo(IArrayOf<IEspECLHelpFile>& helpers, IE
                 thorLog->setProcessName(processName.str());
                 thorLog->setClusterGroup(groupName.str());
                 thorLog->setLogDate(logDate.str());
-                thorLog->setNumberSlaves(numberOfSlaves);
+                thorLog->setNumberSlaves(numberOfSlaveLogs);
```

Comment:
I thought about that. I did not do that because renaming it breaks the existing clients, such as ECLWatch UI. And the NumberSlaves is a member of the ThorLogInfo.